lore/urls.py

```diff
@@ -20,10 +20,12 @@
 from django.contrib import admin
 
 import importer.urls as importer_urls
+import learningresources.urls as lore_urls
 import cas.urls as cas_urls
 
 urlpatterns = [
     url(r'^admin/', include(admin.site.urls)),
+    url(r'^lore/', include(lore_urls)),
```

Comment:
Are we sure this is the URL extension we want?  I know we have already used importer to house views, but if we are putting all the templates in the UI app, don't we also want to have all the views there?  @Ferdi and @pdpinch?  I'm fine with any way we do it, I'm just hoping for architectural guidance on reviews and future PRs with URLs, views, static assets and templates.